src/include/daos/rpc.h

```diff
@@ -213,7 +213,8 @@ int daos_rpc_send_wait(crt_rpc_t *rpc);
 static inline bool
 daos_rpc_retryable_rc(int rc)
 {
-	return daos_crt_network_error(rc) || rc == -DER_TIMEDOUT;
+	return daos_crt_network_error(rc) || rc == -DER_TIMEDOUT ||
+	       rc == -DER_GRPVER;
```

Comment:
Hmm, I did not see where the GRPVER is returned in this patch? in other patches?

Reply:
-DER_GRPVER may be returned by CaRT for CoRPC and IV operations upon group version mismatches. The error code was -DER_MISMATCH, but Xuezhao found it to be potentially conflicting with some DAOS cases, so we changed CaRT to return -DER_GRPVER instead.